src/main/java/org/folio/mr/SecureRequestApplication.java

```diff
@@ -1,13 +1,13 @@
-package org.folio.template;
+package org.folio.mr;
```

Comment:
`mr` does not really say much... Are we planning to have any other sub-packages in `org.folio`? If not, I'd rather place everything in `org.folio` directly.